src/levels.rs

```diff
@@ -16,13 +16,42 @@ pub const TX_16X16: TxfmSize = 2;
 pub const TX_8X8: TxfmSize = 1;
 pub const TX_4X4: TxfmSize = 0;
 
-pub type BlockLevel = u8;
-pub const N_BL_LEVELS: usize = 5;
-pub const BL_8X8: BlockLevel = 4;
-pub const BL_16X16: BlockLevel = 3;
-pub const BL_32X32: BlockLevel = 2;
-pub const BL_64X64: BlockLevel = 1;
-pub const BL_128X128: BlockLevel = 0;
+#[repr(u8)]
+#[derive(Default, Debug, Clone, Copy, PartialEq, Eq)]
+pub enum BlockLevel {
+    BL_8X8 = 4,
+    BL_16X16 = 3,
+    BL_32X32 = 2,
+    BL_64X64 = 1,
+    #[default]
+    BL_128X128 = 0,
```

Comment:
Could you flip the order of these?  `c2rust transpile` transpiles them backwards for some reason, and especially when the numeric value matters it looks very weird backwards.